tests/functional/test_libraries.py

```diff
@@ -607,17 +583,6 @@ def test_pil_PyQt5(pyi_builder):
     import PIL.ImageQt
     """)
 
-@importorskip('PIL.ImageQt')
```

Comment:
in retrospect

I think I should probably add this test back, waiting on reviewer thoughts

Reply:
Move this into a PyQt5 test instead.

Reply:
Oh and update the PIL hook too - maybe `git grep PyQt4` and `git grep PySide`? It'll return all usages in the repo.

Reply:
FYI: Moving into Qt5 test will overlap with #5121 So maybe it is better for this ull-request to just change it here. Then the pull-request merged later should get a conflict.

Reply:
that makes sense @htgoebel